app/views/articles/_single_story.html.erb

```diff
@@ -120,19 +120,19 @@
             <a href="<%= story.path %>" class="crayons-btn crayons-btn--s crayons-btn--ghost crayons-btn--icon-left" data-reaction-count data-reactable-id="<%= story.id %>">
               <%= inline_svg_tag("small-heart.svg", aria: true, width: 24, height: 24, class: "crayons-icon", title: "Reactions") %>
               <%= story.public_reactions_count %>
-              <span class="hidden s:inline">&nbsp;<%= story.public_reactions_count == 1 ? "reaction" : "reactions" %></span>
+              <span class="hidden s:inline">&nbsp;<%= story.public_reactions_count == 1 ? t("core.reaction").downcase : t("core.reaction").downcase.pluralize %></span>
```

Comment:
Please do not hardcode one-or-many type branch into i18n logic, as I commented on another JS file. Also not recommend to use Rails `pluralize` for serious i18n.

Reply:
Thanks! Definitely agree with you, as I commented above, this is far from the final state and all similar logic definitely needs to move out of the view layer. 

My native language is German, so the current limitations of our approach are not lost on me (I think I mentioned this in regards to dates in a comment further up). I speak several others, two of which don't use the Latin alphabet, so I'm very invested in this overall effort of making Forem as good at I18n as it can possibly be. I really hope to see more contributions from you in that regard in the future too!

Reply:
Hi, I think I was overconcerned because I have seen so much badly localized software (and how easily they got). I'm relieved that you have insight how to handle these things correctly. Looking forward to seeing further development!

Reply:
Agreed. I live in a country where a lot of websites are not localized at all or only poorly, so I feel this pain on a daily basis. 😐